main/conceptual/js-programming/hardened-js.md

```diff
@@ -206,23 +204,23 @@ in [JavaScript closures](https://developer.mozilla.org/en-US/docs/Web/JavaScript
 
-[![makeCounter code animation](../../assets/counter-animation.png)](https://pythontutor.com/live.html#code=const%20makeCounter%20%3D%20%28%29%20%3D%3E%20%7B%0A%20%20let%20count%20%3D%200%3B%0A%20%20const%20it%20%3D%20%7B%0A%20%20%20%20incr%3A%20%28%29%20%3D%3E%20%28count%20%2B%3D%201%29,%0A%20%20%20%20decr%3A%20%28%29%20%3D%3E%20%28count%20-%3D%201%29,%0A%20%20%7D%3B%0A%20%20return%20Object.freeze%28it%29%3B%0A%7D%3B%0A%0Aconst%20c1%20%3D%20makeCounter%28%29%3B%0A%0Aconst%20c2%20%3D%20makeCounter%28%29%3B%0Aconsole.log%28c2.incr%28%29%29%3B%0Aconsole.log%28%5Bc1.incr%28%29,%20c2.incr%28%29%5D%29%3B&cumulative=false&curInstr=0&heapPrimitives=nevernest&mode=display&origin=opt-live.js&py=js&rawInputLstJSON=%5B%5D&textReferences=)
+[![makeCounter code animation](../assets/counter-animation.png)](https://pythontutor.com/live.html#code=const%20makeCounter%20%3D%20%28%29%20%3D%3E%20%7B%0A%20%20let%20count%20%3D%200%3B%0A%20%20const%20it%20%3D%20%7B%0A%20%20%20%20incr%3A%20%28%29%20%3D%3E%20%28count%20%2B%3D%201%29,%0A%20%20%20%20decr%3A%20%28%29%20%3D%3E%20%28count%20-%3D%201%29,%0A%20%20%7D%3B%0A%20%20return%20Object.freeze%28it%29%3B%0A%7D%3B%0A%0Aconst%20c1%20%3D%20makeCounter%28%29%3B%0A%0Aconst%20c2%20%3D%20makeCounter%28%29%3B%0Aconsole.log%28c2.incr%28%29%29%3B%0Aconsole.log%28%5Bc1.incr%28%29,%20c2.incr%28%29%5D%29%3B&cumulative=false&curInstr=0&heapPrimitives=nevernest&mode=display&origin=opt-live.js&py=js&rawInputLstJSON=%5B%5D&textReferences=)
 
-## Hardening JavaScript: strict mode
+## Hardening JavaScript: Strict Mode
 
-The first step to hardening JavaScript is that Hardened JavaScript
+The first step to hardening JavaScript is understanding that Hardened JavaScript
 is always in [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode).
 
 [![Subsetting JavaScript](https://raw.githubusercontent.com/endojs/Jessie/HEAD/docs/jessie.png)](https://github.com/endojs/Jessie#subsetting-ecmascript)
 
 One way that you would notice this is if you
-accidentally assign to a frozen property: this will throw a `TypeError`
+try to assign a value to a frozen property: this will throw a `TypeError`
 rather than silently failing.
 
-Important benefits of strict mode include complete encapsulation
-(no `caller` etc.) and reliable static scoping.
+Operating in strict mode yields the important benefits of complete encapsulation
+and reliable static scoping.
```

Comment:
The `caller` example seems particularly relevant and illustrative. `arguments` might be an even better one.

Reply:
I don't know what "no caller" means. Is there a specific example elsewhere in the page that I'm not seeing? In your defense, I haven't read the entire page carefully. In my defense, readers will also often not read the entire page carefully. 
For now, I'll bow to your superior knowledge of the audience; I've reverted the text so that "(no caller etc.)" appears. But when I revisit this topic for proper revision, I'll ping you to get clear what you're getting at & hopefully clarify the text accordingly.

Reply:
"(no caller etc.)" refers to `.caller` in the red part of the diagram just previous:

![Hardened JavaScript _ Documentation](https://user-images.githubusercontent.com/150986/200409296-0fb428b6-d8ad-416a-ac99-cad1a36657d6.png)


We could supplement with a link to [caller](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/caller) in MDN perhaps.